.goreleaser.develop.yaml

```diff
@@ -77,8 +78,8 @@ dockers:
       - "--label=org.opencontainers.image.version={{ .Env.CHAINLINK_VERSION }}"
       - "--label=org.opencontainers.image.url={{ .Env.IMAGE_LABEL_SOURCE }}"
     image_templates:
-      - "{{ .Env.IMAGE_PREFIX }}/{{ .ProjectName }}-develop:develop-root-amd64"
-      - "{{ .Env.IMAGE_PREFIX }}/{{ .ProjectName }}-develop:sha-{{ .ShortCommit }}-root-amd64"
+      - "{{ .Env.IMAGE_PREFIX }}/{{ .ProjectName }}-develop:develop{{ .Env.IMAGE_TAG_SUFFIX }}-root-amd64"
+      - "{{ .Env.IMAGE_PREFIX }}/{{ .ProjectName }}-develop:sha-{{ .ShortCommit }}{{ .Env.IMAGE_TAG_SUFFIX }}-root-amd64"
   - id: root-linux-arm64
     dockerfile: core/chainlink.goreleaser.Dockerfile
```

Comment:
need to parameterize this

Reply:
same w/ amd64 build

Reply:
@ajgrande924 wydm by parameterize? Do you mean have a hyphen separating it? If so, we don't need to worry about it because the suffix includes the hyphen:

https://github.com/smartcontractkit/chainlink/pull/10651/files#diff-5245081c6d881ff6f2e0b51c3b8a606ccd8207cb6c3737ca732437387c6011c0R26

Reply:
ahh so I mean line 84 (references core): 
```yaml
    dockerfile: core/chainlink.goreleaser.Dockerfile
```

Reply:
Resolving this for now because I went a different direction and skipped goreleaser